src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -0,0 +1,144 @@
+    @ApiOperation(value = "Get all products")
+    public List<ProductResponseDto> getAll(@RequestParam(defaultValue = "20")
+                                           @ApiParam(value = "Default value "
+                                                   + "is `20`")
+                                                   Integer count,
+                                           @RequestParam (defaultValue = "0")
+                                           @ApiParam(value = "Default value "
+                                                   + "is `0`")
+                                                   Integer page,
+                                           @RequestParam (defaultValue = "id")
+                                           @ApiParam(value = "Default value "
+                                                   + "is `id`")
+                                                   String sortBy) {
+        Sort sort = Sort.by(sortRequestHandler.parseSortOrders(sortBy));
+        PageRequest pageRequest = PageRequest.of(page, count, sort);
+        return productService.findAll(pageRequest).stream()
+                .map(productDtoMapper::mapToDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/by-price")
+    @ApiOperation(value = "Get all product in price range")
+    public List<ProductResponseDto> getAllByPriceBetween(@RequestParam BigDecimal from,
+                                                         @RequestParam BigDecimal to,
+                                                         @RequestParam(defaultValue = "20")
+                                                         @ApiParam(value = "Default value "
+                                                                 + "is `20`")
+                                                                 Integer count,
+                                                         @RequestParam (defaultValue = "0")
+                                                         @ApiParam(value = "Default value "
+                                                                 + "is `0`")
+                                                                 Integer page,
+                                                         @RequestParam (defaultValue = "id")
+                                                         @ApiParam(value = "Default value "
+                                                                 + "is `id`")
+                                                                 String sortBy) {
+        Sort sort = Sort.by(sortRequestHandler.parseSortOrders(sortBy));
+        PageRequest pageRequest = PageRequest.of(page, count, sort);
+        return productService.findAllByPriceBetween(from, to, pageRequest)
+                .stream()
+                .map(productDtoMapper::mapToDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/inject")
+    @ApiOperation("Inject products for testing")
+    public String injector() {
```

Comment:
```suggestion
    public String inject() {
```
method names should be verbs